.github/workflows/longevity_test.yml

```diff
@@ -230,6 +230,62 @@ jobs:
 
 ##########################################################################################################################################
 
+# Export OneDrive Test
+      - name: OneDrive Export test
+        run: |
+          set -euo pipefail
+          echo -e "\Export OneDrive test\n" >> ${CORSO_LOG_FILE}
+
+          echo -e "\Export OneDrive test - first entry\n" >> ${CORSO_LOG_FILE}
+          ./corso backup list onedrive 2>/dev/null | tail -n+2 | head -n1 | awk '{print $1}' |
```

Comment:
does this and the last backup export command assume there's at least one backup? The other alternative is to write some wrapper code for the corso SDK. Not sure if that would be easier when it comes to logging/output, but may allow for greater control of backup selection

I'm alright with suggesting an SDK wrapper if that makes more sense for this use-case because sanity testing already gives us coverage ensuring the command works

Reply:
> does this and the last backup export command assume there's at least one backup?

Yup, I'm assuming that is an OK assumption to make as we are doing a backup right before this.

> but may allow for greater control of backup selection

Any specific requirement that you have in mind for selecting backups?

Reply:
ah good point about the backup before this. I guess I should pay more attention to the unchanged lines sometimes haha. I think mostly just edge case handling when it comes to backup selection, but probably not worth worry over too much. I think worst case we'll just end up exporting the same backup 2x if it's the only backup at the time